be/src/pipeline/exec/partitioned_aggregation_sink_operator.cpp

```diff
@@ -170,6 +170,9 @@ Status PartitionedAggSinkOperatorX::sink(doris::RuntimeState* state, vectorized:
         if (revocable_mem_size(state) > 0) {
             RETURN_IF_ERROR(revoke_memory(state));
         } else {
+            for (auto& partition : local_state._shared_state->spill_partitions) {
```

Comment:
There are some problems:
1. this API will write disk, it may blocking pipeline thread.
2. If eos and revoable mem size == 0, why do we need finish current spilling, since there are no data.